docs/knowledgebase/getting-started/glossary.md

```diff
@@ -390,6 +503,32 @@ The provenance of a dispatched function call into the runtime. Can be customised
 
 ---
 
+## Parachain
+
+A parachain is a [blockchain](#blockchain) that encapsulates capabilities that are specific to a
+domain or application. The term "parachain" was defined in
+[the Polkadot Whitepaper](https://polkadot.network/PolkaDotPaper.pdf) and refers to the blockchains
+that derive shared infrastructure and security from a "[relay chain](#relay-chain)".
+[Rococo](#rococo) is the Polkadot Network's parachain test network.
+
+## Pallet
+
+A module that encapsulates a set of related capabilities that is used to compose a
+[FRAME](#frame)-based [runtime](#runtime).
+
+## Polkadot Network
+
+The [Polkadot Network](https://polkadot.network/) is a next-generation platform that implements a
+heterogenous ("chain-of-chains") architecture. The Polkadot Network is a Substrate-based
+[blockchain](#blockchain) that serves the role of "[relay chain](#relay-chain)", a central hub in a
+heterogenous blockchain network that supports the other chains (the "[parachains](#parachain)") by
+providing shared infrastructure and security. The initial stages of
+[Polkadot's multi-phase launch process](https://polkadot.network/explaining-the-polkadot-launch-process/)
+do not include support for parachains; [Rococo](#rococo) is the Polkadot Network's parachain test
+network. Although the Polkadot Network was maintained by
+[Parity Technologies](https://www.parity.io/) when it initially launched, it is already controlled
+by [secure, decentralized, collective governance](#council).
```

Comment:
Agree, this is not a good definition of Polkadot. And I don't see how the launch process is relevant at all.

Reply:
Joe, that is not a helpful comment. Can you please provide some constructive criticism?

Reply:
I updated this definition but left a note about the launch process because I don't want people to read this definition and think that Polkadot offers capabilities that it does not currently support. Happy to hear suggestions for way to improve :+1: